Arena.toml

```diff
@@ -18,1099 +18,1374 @@ filters = ["/template/task-templates.wdl"]
 [[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
 message = "ww-fastq-to-cram.wdl:129:5: note[MatchingParameterMeta]: task `FastqToUnmappedBam` has an extraneous parameter metadata key named `unmapped_bam`"
+permalink = "https://github.com/getwilds/ww-fastq-to-cram/blob/7dd6c4c7c8c9fc6dcd4eb5ff8f3262cd26d5e7cc/ww-fastq-to-cram.wdl"
```

Comment:
Should the line information be embedded in the permalink? I would think so.

Reply:
From the PR description:

> Note: these permalinks could contain a line number for an even better user experience, but I couldn't figure that out 😅

So the answer is yes, I was just stumped on how to do that

Reply:
An `#L<NUM>` anchor can be used on the URL to bring you to the line.

And I think we can utilize [line_index](https://docs.rs/codespan-reporting/latest/codespan_reporting/files/trait.Files.html#tymethod.line_index) to go from span start offset to line.

Reply:
The problem I'm having is that all the `Diagnostics` get converted into plain `String`s [here](https://github.com/stjude-rust-labs/wdl/blob/feat/gauntlet-urls/wdl-gauntlet/src/lib.rs#L225). I would either have to retrieve the line numbers from the raw string (possible but messy) or rewrite significant portions of this code to pass around `Diagnostics` instead of `Strings`.